tests/api_tests/nodes/test_views.py

```diff
@@ -2447,8 +2447,9 @@ def test_return_public_node_pointers_logged_out(self):
         res_json = res.json['data']
         assert_equal(len(res_json), 1)
         assert_equal(res.status_code, 200)
-        assert_in(res_json[0]['attributes']['target_node_id'], self.public_pointer_project._id)
         assert_equal(res.content_type, 'application/vnd.api+json')
+        url = res.json['data'][0]['relationships']['target_node']['links']['related']['href']
+        assert_in('/{}nodes/{}/'.format(API_BASE, self.public_pointer_project._id), url)
```

Comment:
The port difference isn't too surprising- after all, webtest doesn't run by making requests to your `apiserver`. (if you don't believe me, try running your tests without `apiserver` running- [just like Travis](https://github.com/abought/osf.io/blob/develop/.travis.yml#L88-88)!)

Webtest internally has its own system of requests for testing, which appears to [default to port 80](http://stackoverflow.com/questions/2774249/python-webtest-port-configuration). You could change it, but really not worth the headache.